scripts/docker/gatkbase/build_docker_base.sh

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env bash
+
+# Have script stop if there is an error
+set -e
+
+
+REPO=broadinstitute
+PROJECT=gatk
+VERSION=1.0
+FULL_PATH=${REPO}/${PROJECT}:gatkbase-${VERSION}
+
+#################################################
+# Parsing arguments
+#################################################
+while getopts "ph" option; do
+	case "$option" in
+	    h) IS_HELP=true ;;
+		p) IS_PUSH=true ;;
+	esac
+done
+
+if [ -n "$IS_HELP" ]; then
+	printf "Build the GATK4 base image.\n \
+Usage: %s: [-p] [-h] \n \
+Optional arguments:  \n \
+-p \t push image to docker hub once complete.  This will use the GITHUB_TAG in dockerhub as well. \n \n" $0
+	exit 1
+fi
+
+# Make sure sudo or root was used.
+if [ "$(whoami)" != "root" ]; then
```

Comment:
Why we need to be root, is it really necessary? ca we use sudo instead whenever privileges are needed?